bundle.md

```diff
@@ -10,11 +10,11 @@ The definition of a bundle is only concerned with how a container, and its confi
 A Standard Container bundle contains all the information needed to load and run a container.
 This MUST include the following artifacts:
 
-1. `config.json`: contains configuration data.
+1. <a name="containerFormat01" />`config.json`: contains configuration data.
```

Comment:
Does it make sense to actually not be hyphenated so we can distinguish them from github/pandoc anchors?